listener/upload.go

```diff
@@ -0,0 +1,153 @@
+package listener
+
+import (
+	"app/base/database"
+	"app/base/models"
+	"app/base/utils"
+	"context"
+	"crypto/sha256"
+	"encoding/hex"
+	"encoding/json"
+	"github.com/RedHatInsights/patchman-clients/inventory"
+	"github.com/RedHatInsights/patchman-clients/vmaas"
+	"github.com/antihax/optional"
+	"github.com/segmentio/kafka-go"
+	"time"
+)
+
+func uploadHandler(m kafka.Message) {
+	var event PlatformEvent
```

Comment:
How can I test it? Is there any script or guide to run all workflow using docker-compose?

Reply:
Not yet, we'll need to either extend Unit tests to run with platform, or introduce separate tools for testing whole workflows ...